### ISSUE TITLE
doesn't seem to be handling non-existent commands

### ISSUE THREAD

**Issue by dmalan** [cs50/help50#251]
doesn't seem to be handling non-existent commands (e.g., `help50 foo` where `foo` is not a command)?

```
$ help50 foo
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.9/bin/help50", line 126, in <module>
    main()
  File "/home/travis/virtualenv/python2.7.9/bin/help50", line 43, in main
    proc = ptyprocess.PtyProcess.spawn(sys.argv[1:])
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/ptyprocess/ptyprocess.py", line 208, in spawn
    'executable: %s.' % command)
ptyprocess.ptyprocess.FileNotFoundError: The command was not found or was not executable: foo.
```

CC @kzidane

**Comment by dmalan** [comment]
@crossroads1112 expected behavior:

<img width="828" alt="screen shot 2017-06-27 at 5 29 46 pm" src="https://user-images.githubusercontent.com/788678/27610951-55c4adea-5b5e-11e7-8578-53d7597fa1f2.png">

**Comment by crossroads1112** [comment]
Addressed in #254
